crates/lsp/src/lib.rs

```diff
@@ -67,6 +68,15 @@ impl<L: LSPLang> Backend<L> {
 
 const FALLBACK_CODE_ACTION_PROVIDER: Option<CodeActionProviderCapability> =
   Some(CodeActionProviderCapability::Simple(true));
+
+trait AstGrepActionKind {
+  const SOURCE_FIX_ALL_AST_GREP: CodeActionKind;
+}
+
+impl AstGrepActionKind for CodeActionKind {
+  const SOURCE_FIX_ALL_AST_GREP: CodeActionKind = CodeActionKind::new("source.fixAll.ast-grep");
```

Comment:
QQ: why do we need a new fixAll variant for ast-grep? Should it be implemented as fixAll?

Reply:
```json
 "editor.codeActionsOnSave": {
    "source.fixAll.eslint": "explicit"
  },
```
I think this is a specification of vscode.
https://github.com/microsoft/vscode-eslint/blob/c7ab1ded33213a43ce79b5caa4372ffa4ac98dc3/server/src/eslintServer.ts#L590-L591
https://github.com/ast-grep/ast-grep-vscode/issues/67#issuecomment-1943096215

In this way, they can also be defined in order of execution.
```json
 "editor.codeActionsOnSave": [
 	 "source.organizeImports", "source.fixAll.eslint",  "source.fixAll.ast-grep"
 ]
```